SixTrack/sixtrack.s90

```diff
@@ -10750,8 +10764,8 @@ subroutine daten
       iskew=0
       preda=c1m38
 !hion
-      zz0 = 1
-      aa0 = 1
+      zz0 = one
```

Comment:
these are integers, so using "1" is totally ok. "one" is a floating point variable and there will be a cast taking place with this.